czespressif/config.py

```diff
@@ -100,40 +103,57 @@ def known_types(self) -> list[CommitType]:
 
     @property
     def incremental(self) -> bool:
-        return '--incremental' in sys.argv
+        incr = False  # Default is full changelog with headers
+
+        # Check if the CLI arguments contain a version or range ('v1.2.3' or '1.2.3..4.5.6')
+        version_or_range = any(re.match(r'^\d+\.\d+\.\d+(\.\.\d+\.\d+\.\d+)?$', arg) for arg in sys.argv) or any(arg.startswith('v') for arg in sys.argv)
+        if version_or_range:
+            incr = True
+
+        # Check if the CLI arguments contain the incremental flag ('--incremental')
+        incremental_flag = '--incremental' in sys.argv
```

Comment:
You can either get rid of the unnecessary variables `version_or_range`, `incremental_flag` and `bump` like: 

```py
if '--incremental' in sys.argv:
    incr = True
```

or set `incr` at one place at the end of the function:

```py
if version_or_range or incremental_flag or bump:
    incr = True
```

Reply:
@radimkarnis Thank you ... this was deliberately split for dev and testing, now its safe to refactor as you suggest.